src/renderer/components/swap/Swap.utils.ts

```diff
@@ -156,7 +156,7 @@ export const getSwapData = ({
  */
 export const getSwapLimit = (swapResultAmountMax1e8: BaseAmount, slipTolerance: SlipTolerance): BaseAmount => {
   const swapLimit: BaseAmount = swapResultAmountMax1e8.times(1.0 - slipTolerance * 0.01)
-  const swapLimitWithIdentifier: number = +swapLimit.amount().toString().slice(0, -3).concat(ASGARDEX_SWAP_IDENTIFIER)
+  const swapLimitWithIdentifier = +swapLimit.amount().toString().slice(0, -3).concat(ASGARDEX_SWAP_IDENTIFIER) - 1000
```

Comment:
Pls add tests for lower `swapLimit` (< `999` , e.g. `100`, `200` etc.) to see `swapLimitWithIdentifier` is still `999`

Reply:
Those are really rare cases and if the user deals with those values, then setting limit would probably be incorrect in this case.

Reply:
But I've updated the code in order to handle these cases.